src/transpiling/jsx_precompile.rs

```diff
@@ -737,18 +884,14 @@ impl JsxPrecompile {
     strings: &mut Vec<String>,
     dynamic_exprs: &mut Vec<Expr>,
   ) {
-    for child in children.iter() {
+    let (normalized_children, _text_count, _serializable_count) =
+      merge_serializable_children(children);
+
+    for child in normalized_children.iter() {
```

Comment:
Since we pre-normalize, I think this can just be `for child in normalized_children {` now. Then any cloning below can be removed.

Reply:
Thanks 👍